clang/test/SemaCXX/warn-unused-private-field.cpp

```diff
@@ -20,6 +20,17 @@ class SpaceShipDefaultCompare {
   int operator<=>(const SpaceShipDefaultCompare &) const = default;
 };
 
+class UnusedConstPrivateField {
+ public:
+  UnusedConstPrivateField() : unused_(0) {}
+ private:
+  const int unused_; // expected-warning{{private field 'unused_' is not used}}
+};
+
+class FriendEqDefaultCompare {
```

Comment:
I would emphasize with the name that this class has no relation to the previous one, because that is what the bug is about.

```suggestion
class UnrelatedFriendEqDefaultCompare {
```